src/algebra/order/monoid_lemmas_zero_lt.lean

```diff
@@ -294,19 +294,19 @@ section partial_order
 variables [partial_order α]
 
 @[priority 100] -- see Note [lower instance priority]
-instance pos_mul_strict_mono.to_pos_mul_mono [pos_mul_strict_mono α] : pos_mul_mono α :=
+instance pos_mul_mono_of_pos_mul_strict_mono [pos_mul_strict_mono α] : pos_mul_mono α :=
```

Comment:
The old names look better to me, they match things like `group.to_monoid`

Reply:
I rename them to new names because there are some lemmas and instances like `contravariant_mul_lt_of_covariant_mul_le`.